examples/sample-foundry-project/cannonfile.toml

```diff
@@ -1,4 +1,4 @@
-name = "greeter-foundry"
+name = "greeter"
 version = "<%= package.version %>"
 description = "Simple project to verify the functionality of cannon"
 keywords = ["sample", "greeter"]
```

Comment:
Would be good to add an action on this or other of the cannonfiles used on the test with Inifinex's case, was is with something like `<%= settings.smth['VALUE_' + chainId]`?